docs/src/content/docs/zh-cn/resources/community-content.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 社区内容
+description: 发现社区制作的指南、文章和视频，以帮助你学习和构建 Starlight！
+---
+
+:::tip[添加你自己的！]
+你是否制作过关于 Starlight 的内容？
+创建一个 PR 来添加链接到此页面！
+:::
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+## 文章和评论
+
+以下是一系列帖子和文章，以了解有关 Starlight 和其他人的经历的更多信息：
+
+<CardGrid>
+	<LinkCard
+		href="https://devm.io/open-source/starlight-astro"
+		title="使用 Starlight 生成静态站点"
+		description="“在设计组件时，没有太大或太小的想法”——对 Starlight 负责人 Chris Swithinbank 的采访"
+	/>
+	<LinkCard
+		href="https://frontendatscale.com/blog/hybrid-frontend-architecture/"
+		title="使用 Astro 和 Starlight 的混合前端架构"
+		description="Maxi Ferreira 和 Ben Holmes 使用 Starlight、TinaCMS 和具有身份验证功能的交互式 API Playground 构建了一个文档站点。"
+	/>
+	<LinkCard
+		href="https://www.olets.dev/posts/comparing-docs-site-builders-vuepress-vs-starlight/"
+		title="比较文档网站构建工具：VuePress 与 Starlight"
+		description="这两个框架如何衡量？"
+	/>
+</CardGrid>
+
+## 方案和指南
+
+方案通常是短小的、专注的指南，指导读者完成特定任务的工作示例。通过按照逐步说明完成，方案是向 Starlight 项目添加新功能或行为的绝佳方式！其他指南可能会解释与内容领域相关的概念，例如使用图像或使用 MDX。
+
+浏览由 Starlight 用户维护的社区内容：
+
+<CardGrid>
+	<LinkCard
+		href="https://www.webpro.nl/scraps/versioned-docs-with-starlight-and-vercel"
+		title="Starlight & Vercel 的版本化文档"
+		description="为项目的每个主要版本实现单独版本的文档的指南"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-heading-links"
+		title="添加指向 Starlight 标题的链接"
+		description="使用 rehype 插件共享文档特定部分链接的指南"
+	/>
+	<LinkCard
+		href="https://blog.otterlord.dev/posts/starlight-sponsors/"
+		title="将赞助商添加到你的 Starlight 网站"
+		description="在文档侧边栏中实现自定义赞助商组件的指南"
+	/>
+	<LinkCard
+		href="https://hideoo.dev/notes/starlight-og-images"
+		title="将 Open Graph 图像添加到 Starlight"
+		description="A guide to generating social images and the corresponding meta tags for your pages"
```

Comment:
I think this line was missed for translation.

Reply:
Fixed it.